client/src/pages/Home.js

```diff
@@ -19,6 +20,7 @@ export function Home() {
 			<div className="whats-new">
 				<WhatsNewCard />
 				<WhatsNewCard />
+				<Description />
```

Comment:
I'm not sure this is the right place for this? Also is `<Description />` a good name for this component?